packages/user-operation-controller/CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [16.0.0]
+
+### Added
+
+- Types for `bridge` and `bridgeApproval`. ([#4714](https://github.com/MetaMask/core/pull/4714))
```

Comment:
This reads like we're defining new TypeScript types. Maybe we could be a little clearer on what's being updated:

```suggestion
- Added new entries `bridge`, `bridgeApproval` to the `TransactionType` enum ([#4714](https://github.com/MetaMask/core/pull/4714))
```